activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+*   Calling `iso8601` on negative durations retains the negative sign on individual 
+    digits instead of prepending it.
```

Comment:
Let's communicate why we're doing this: so we can interoperate with PostgreSQL, which prefers negative signs for each component. And that we retain compatibility with other iso8601 parsers which support leading negatives as well as negatives per component.

Reply:
Last call for Rails 6.1.0 release @vipulnsward /cc @rafaelfranca